internal/jasmine_node_test/jasmine_node_test.bzl

```diff
@@ -41,10 +41,12 @@ def jasmine_node_test(name, srcs = [], data = [], deps = [], **kwargs):
   all_data += [Label("//internal/jasmine_node_test:jasmine_runner.js")]
   all_data += [":%s_devmode_srcs.MF" % name]
   entry_point = "build_bazel_rules_nodejs/internal/jasmine_node_test/jasmine_runner.js"
+  tags = ["ibazel_notify_changes"]
 
   nodejs_test(
       name = name,
       data = all_data,
+      tags = tags,
```

Comment:
this prevents the user adding other tags on the target - should amend this macro to have argument `tags = []` and add the ibazel one to the given ones

Reply:
fixed